### ISSUE TITLE
Massage the opam requires at import time.

### ISSUE THREAD

**Issue by jordwalke** [esy/pesy#76]
When we do:

```
        "YoJson = require('@opam/yojson')",
```

Then dune will say:
```
      Fl_package_base.No_such_package("opam--yojson", "")
```

because opam packages weren't created with pesy.

I think we should probably special case opam package names to not assume they abide by the convention? Or is that not a good idea?

**Comment by prometheansacrifice** [comment]
Right. Otherwise, the workflow currently expects users to look up 'yojson' in the list returned by `esy pesy ls-libs`.

**Comment by prometheansacrifice** [comment]
Closed by #91
